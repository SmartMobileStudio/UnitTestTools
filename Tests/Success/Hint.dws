procedure Dummy; deprecated;
begin
end;

  Dummy;
Dummy;