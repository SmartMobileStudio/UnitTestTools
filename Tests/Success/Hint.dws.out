SmartMS command-line compiler, (c) 2016 The Smart Company AS
Building Tests\Success\Hint.dws
[INFO] Building project ''...
[INFO] Compiling...
Warning: "Dummy" has been deprecated [line: 5, column: 3]
Warning: "Dummy" has been deprecated [line: 6, column: 1]
[INFO] Generating JavaScript...
[INFO] Linking...
